sync-orders-to-bigQuery/decrypt-helper.js

```diff
@@ -0,0 +1,40 @@
+const GoogleKMS = require('@google-cloud/kms').KeyManagementServiceClient
```

Comment:
Would it not be easier to reduce the code and use `gcloud` natively without Serverless Framework so users can use the CLI to set their `ENV` variables?